com.github.libresprite.LibreSprite.yaml

```diff
@@ -0,0 +1,41 @@
+app-id: com.github.libresprite.LibreSprite
+runtime: org.freedesktop.Platform
+runtime-version: 1.6
+sdk: org.freedesktop.Sdk
+command: libresprite
+rename-appdata-file: libresprite.appdata.xml
+rename-desktop-file: libresprite.desktop
+rename-icon: libresprite
+finish-args:
+  - --socket=wayland
```

Comment:
Does it use SDL2/gtk3? Otherwise I guess it's wrong and should have just X11.

Reply:
Right, removed it.